requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests == 2.18.4
```

Comment:
You probably don't need to pin the version number so that it can take the last version automatically.